docs/sdss/sdss.rst

```diff
@@ -4,6 +4,16 @@
 SDSS Queries (`astroquery.sdss`)
 ********************************
 
+Default Data Release
+====================
+
+The default data release is set to Data Release 17 (DR17), which is
+the final data release of the Sloan Digital Sky Survey IV.  DR17
+contains new optical and infrared spectra from both Apache Point
+Observatory and Las Campanas Observatory. Previously released
+integral-field datacubes and maps, stellar library spectra, as well as
+images, are also included in DR17.  Users may select alternate DR's.
```

Comment:
How can a different data release be selected? Documentation is not helpful is it just says that something can be done, but neither describes nor provides examples how.

Reply:
Sorry, first timer.  In the core routine of the sdss package, the various query procedures allow for the data release parameter to be specified (with the default DR set initially).  This has never been documented properly with examples (though seems straightforward for user specification).  The easiest thing would be to remove the phrase now and then, take some time to generate an example.  Sound alright?

Reply:
The fact that changing the data release is not documented is a shortcoming that should be fixed, but I would tend to agree that it would be beyond the scope of a pull request that simply updates the default. But perhaps it would be better to remove the entire section because only removing the last sentence might leave the wrong impression that DR17 is the only option.

Reply:
I spoke about this with @weaverba137.  And no, it is important to note what the default data release is and what it contains.  This omission (by astroquery) is what lead to the initial issue request as users were confused as to why DR14 was employed by astroquery as opposed to the most recent DR from SDSS (DR17).  The language can be massaged accordingly, but users do have some inherent responsibility to understand something about the database/dataset they are querying (and should perform some due diligence). If a user is employing SDSS data, there is a high probability that they understand that it has been operating since 2000 and has issued muliple DR's over the years.

Reply:
I think the language @jsobeck is suggesting should remain, because it opens the door to astroquery eventually getting the functionality to access these other SDSS data sets.

More comments below.